source/php/Component/Block/block.json

```diff
@@ -26,6 +26,20 @@
         "dateBadge": "Display date as a badge.",
         "icon": "An array with the same specification as the icon component"
     },
+    "types": {
+        "heading": "string",
+        "content": "string|array",
+        "meta": "string",
+        "secondaryMeta": "string|array",
+        "image": "array|boolean",
```

Comment:
array should be enough since by default there is no image. If you want an image you pass an array otherwise do nothing.